api/server/server.go

```diff
@@ -270,7 +274,9 @@ func createGRPCServer(opts []grpc.ServerOption, webProxyAddr string) (*grpc.Serv
 		grpcweb.WithOriginFunc(func(origin string) bool {
 			return true
 		}),
+		grpcweb.WithAllowedRequestHeaders([]string{"*"}),
 		grpcweb.WithWebsockets(true),
+		grpcweb.WithWebsocketPingInterval(WSPingInterval),
```

Comment:
Just adding the WS ping stuff in here in case we decide to use a WS transport from the client at some point.